iphone/Maps/Core/TextToSpeech/TTSTester.mm

```diff
@@ -1,9 +1,10 @@
 #import "TTSTester.h"
 
-#include <CoreApi/Framework.h>
 #include "LocaleTranslator.h"
 #include "MWMTextToSpeech.h"
 
+#include "base/logging.hpp"
```

Comment:
@kirylkaveryn how Maps target is linked now with the framework and C++ core? Could it be that Maps target statically links with C++ libs, and Core framework also statically links with C++ libs, so two binaries have a similar copy of C++ under the hood?

Reply:
CoreApi framework doesnt have any embeeded libraries or frameworks. All the cpp libraries are statically linked only to the Map tareget.

Reply:
@kirylkaveryn what was the goal of the CoreApi then in your opinion? Just to move all ObjC code there and leave the app in plain swift? What are pros and cons of this approach?